src/app/(pages)/feedback/page.jsx

```diff
@@ -1,72 +1,86 @@
 /* eslint-disable no-unused-vars */
-import React from "react";
+
+"use client";
+
+import React, { useState } from "react";
 import styles from "./feedback.module.scss";
 
 const Feedback = () => {
+  const [selectedOption, setSelectedOption] = useState("");
+
+  const handleRadioChange = (e) => {
+    setSelectedOption(e.target.value);
+  };
+
   return (
     <div className={styles.boxContent}>
       <h1 className={styles.Content}> Your feedback matter to us</h1>
       <div className={styles.boxContainer}>
-        <div className={styles.content1}>How was the consultation?</div>
+        <h2 className={styles.content1}>How was the consultation?</h2>
         <div className={styles.boxes}>
           <div className={styles.boxSelection}>
             <div className={styles.option1}>
               <h4>Loved it!</h4>
               <input
                 type="radio"
-                name=""
+                name="feedbackOption"
```

Comment:
Instead of "feedbackOption", it would be "Loved it!" , "Great" etc.

Reply:
Also, select the radio button when clicked on the larger box not the individual radio.